### ISSUE TITLE
slide gesture

### ISSUE THREAD

**Issue by li-rh** [kiwibrowser/src.next#676]
could you add a gesture to back or ahead website? like the old version in 2019 year

**Comment by MooShuOrc** [comment]
Settings > Accessibility > Edge swipe

**Comment by li-rh** [comment]
however, it does not work after that version.
i sincerely hope you can check it,or tell me what's wrong with mine.



---Original---
From: ***@***.***&gt;
Date: Fri, Aug 19, 2022 11:54 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [kiwibrowser/src.next] slide gesture (Issue #676)




 
Settings &gt; Accessibility &gt; Edge swipe

**Comment by MooShuOrc** [comment]
It works for me on the current version (two hours old). If it's not working for you, you should provide more info, i.e. device model, Android version.

**Comment by li-rh** [comment]
my mobile phone is realme x1, and android version is 11.
by the way, the old version(2019) of kiwibrowser can work on my phone.



---Original---
From: ***@***.***&gt;
Date: Fri, Aug 19, 2022 12:18 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [kiwibrowser/src.next] slide gesture (Issue #676)




 
It works for me on the current version (two hours old). If it's not working for you, you should provide more info, i.e. device model, Android version.

**Comment by MooShuOrc** [comment]
Does your OS have some kind of gestures built in? Like "gesture navigation" or something similar? That can interfere with Kiwi's gestures. Just an idea. I'm not a developer.

**Comment by li-rh** [comment]
yes, it is reason, but i hope can make a bit promotion to avoid this situation. i have been familiar with gesture navigation of my phone. hopeing author can resolve the issue.
in the end, thanks for your reply very much!



---Original---
From: ***@***.***&gt;
Date: Fri, Aug 19, 2022 12:32 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [kiwibrowser/src.next] slide gesture (Issue #676)




 
Does your OS have some kind of gestures built in? Like "gesture navigation" or something similar? That can interfere with Kiwi's gestures. Just an idea. I'm not a developer.

**Comment by JingMatrix** [comment]
Please refer to the [official doc](https://developer.android.com/develop/ui/views/touch-and-input/gestures/gesturenav#kotlin) to support gestures for android 11+.
